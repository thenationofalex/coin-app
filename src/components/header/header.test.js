import React, {Component} from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

import Header from './index'
 
test('Header renders', () => {
  const wrapper = shallow(
    <Header />
  );
 
  expect(toJson(wrapper)).toMatchSnapshot();
});
