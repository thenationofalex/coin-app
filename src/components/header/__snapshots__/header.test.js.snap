// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders 1`] = `
<React.Fragment>
  <Styled(header)>
    <Styled(div)>
      <Link
        replace={false}
        to="/"
      >
        <Styled(h1)>
          blockbid
        </Styled(h1)>
      </Link>
    </Styled(div)>
  </Styled(header)>
</React.Fragment>
`;
